core/arch/x86/decode_table.c

```diff
@@ -6785,6 +6785,10 @@ const instr_info_t evex_W_extensions[][2] = {
     {OP_vpxord, 0x660fef10, "vpxord", Ve, xx, KEw, He, We, mrm|evex, x, END_LIST},
     {OP_vpxorq, 0x660fef50, "vpxorq", Ve, xx, KEb, He, We, mrm|evex, x, END_LIST},
   },
+  {    /* evex_W_ext 45 */
+    {OP_vpmulld,  0x66384018, "vpmulld",   Ve, xx, KEw,He,We, mrm|evex|reqp, x, END_LIST},
+    {OP_vpmullq,  0x66384058, "vpmullq",   Ve, xx, KEb,He,We, mrm|evex|reqp, x, END_LIST},
```

Comment:
I spot-checked this one and something seems off.

I took your branch and set VERBOSE to 1 in ir_x86.c.  From the output there is this line:
```
0x000055a3261f4f0e  62 e2 f5 47 40 41 37 vpmullq {%k7} %zmm17 0x37(%rcx)[64byte] -> %zmm16
```

(These vex and evex encodings make the stored 0x66384058 not very useful: for regular opcodes it makes it easy to see what the raw encoding generally looks like, but not here...not sure what to do about it.)

I fed those raw bytes to two decoders and they think the memory disp is different:

```
$ echo 0x62 0xe2 0xf5 0x47 0x40 0x41 0x37 0x90 0x90 | /usr/bin/llvm-mc -arch x86-64 --disassemble 
	.text
	vpmullq	3520(%rcx), %zmm17, %zmm16 {%k7}
	nop
	nop

$ echo 0x62 0xe2 0xf5 0x47 0x40 0x41 0x37 0x90 0x90 | /extsw/pkgs/disasm/capstone/build/capstone -x86 -
+0x0000  62e2f547404137   vpmullq	zmm16 {k7}, zmm1, zmmword ptr [ecx + 0xdc0]
+0x0007  90   nop	
+0x0008  90   nop	
```

Reply:
run arm tests

Reply:
Ah ok yes, this is probably due to the compressed displacement in AVX-512. The displacement in such instructions is scaled by the vector size in order to make it more useful. This is not supported yet.

Reply:
55 * 64 = 0xdc0.

Reply:
Is this the only thing you hit in your local testing comparing to other decoders?  Or is there a list of missing things I could consult if other spot-checks see differences, to know whether it's a known difference?

Reply:
This is the only thing I know of. But I haven't tested the broadcast or rounding bits yet.

Reply:
I filed #3638 on turning this method of comparing api.ir disassembly to an external decoder into a regression test.